### DISCUSSION TITLE
How to download a previous version?

### DISCUSSION THREAD

**Discussion by Sinfamy** [MicrosoftEdge/WebView2Feedback#2007] [Q&A]
Hello, I'm looking for version "95.0.1020.53" of the x64 Evergreen Standalone Installer.
There is the Fixed Version, but I'd like the installer.
If there is no archive anywhere, is there at least a flag / parameter I can use on the Evergreen Bootstrapper to download the specific version I want?
Thanks.

**Comment by westinyang** [comment]
I have archived WebView2 Fixed Version Runtime since the beginning of 2021, and currently I have collected versions 88 ~ 104. I created a repository and uploaded the archived files through Releases, hoping to help everyone!
https://github.com/westinyang/WebView2RuntimeArchive
